File: src/java/MySQL_serial.java
import java.sql.*;
import java.util.Locale;

public class MySQL_serial {
  public static void main(String[] args) {
    Connection conn;
    Statement stmt;
    String query = "";
    int i;
    double x, prob;
    Locale.setDefault(new Locale("en", "US"));
    try {
      conn = DriverManager.getConnection("jdbc:mysql://localhost/gaussian?allowMultiQueries=true", "root", "root123");
      stmt = conn.createStatement();
      stmt.executeUpdate("DROP TABLE IF EXISTS java_serial; CREATE TABLE java_serial(id INTEGER AUTO_INCREMENT " +
      "PRIMARY KEY, z_score FLOAT NOT NULL, cumulative_distribution FLOAT NOT NULL)");
      for (i = -500; i <= 500; i++) {
        x = (double) i / 100.0;
        prob = Gaussian.gaussianCDF(0, 1, x);
        query = String.format("INSERT INTO java_serial(z_score, cumulative_distribution) VALUES (%.2f, %f);", x, prob);
        stmt.executeUpdate(query);
      }
    } catch (Exception e) {
      System.err.println(e.getMessage());
    }
  }
}

File: src/cs/mysql_serial.cs
using System;
using System.Globalization;
using MySqlConnector;

namespace Calc
{
  class Program
  {
    static void Main(string[] Args)
    {
      MySqlConnection conn;
      MySqlCommand cmd;
      MySqlConnectionStringBuilder builder = new MySqlConnectionStringBuilder
      {
        Server = "localhost",
        Database = "gaussian",
        UserID = "root",
        Password = "root123",
        SslMode = MySqlSslMode.Required,
      };
      int i;
      double x, prob;
      CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
      try
      {
        
        conn = new MySqlConnection(builder.ConnectionString);
        conn.Open();
        cmd = conn.CreateCommand();
        cmd.CommandText = "DROP TABLE IF EXISTS cs_serial; CREATE TABLE cs_serial(id INTEGER AUTO_INCREMENT " +
        "PRIMARY KEY, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL)";
        cmd.ExecuteNonQuery();
        for (i = -500; i <= 500; i++)
        {
          x = (double)i / 100.0;
          prob = Gaussian.GaussianCDF(0, 1, x);
          cmd.CommandText = $"INSERT INTO cs_serial(z_score, cumulative_distribution) VALUES ({x:0.00}, {prob:0.000000});";
          cmd.ExecuteNonQuery();
        }
        conn.Close();
      }
      catch (System.Exception e)
      {
        Console.WriteLine(e.Message);
      }
    }
  }
}
